GOVERNANCE.md

```diff
@@ -0,0 +1,131 @@
+# Fn Project Governance
+
+This document describes the rules and governance of the project. It is meant to be followed by all the developers of the project and the Fn Project community. Common terminology used in this governance document are listed below:
+  * Team members: members of the [Fn Team group on forums.fnproject.io](https://forums.fnproject.io/groups/FnTeam).
+  * Maintainers: Maintainers lead an individual project or parts thereof.
+  * Projects: A single repository in the [Fn Project GitHub organization](https://github.com/fnproject).
```

Comment:
If "Projects" is plural, the description should probably be "The individual repositories in..." - alternatively, make it "Project".

Reply:
It makes most sense to me for the definitions to define a single thing: member, maintainer, project.